docs/examples/modules/DropdownExample/SelectionWithIcons.example.vue

```diff
@@ -0,0 +1,28 @@
+<template lang="html">
+  <sui-dropdown
+    placeholder="Gender"
+    selection
+    :options="options"
+    v-model="current"
+  />
+</template>
+
+<script>
+export default {
+  name: 'SelectionWithIcons',
+  data() {
+    return {
+      current: null,
+      options: [{
+        text: 'Male',
+        icon: 'male',
+        value: 1,
+      }, {
+        text: 'Female',
+        icon: 'female',
+        value: 2,
+      }],
+    };
+  },
+};
+</script>
```

Comment:
Could you use the same example of the original doc? https://semantic-ui.com/modules/dropdown.html#icon